### DISCUSSION TITLE
✅ 🔗 Add a "Content Max-Width" setting that can accept vw and % (Duplicate)

### DISCUSSION THREAD

**Discussion by provenpaige** [elementor/elementor#20346] [Feature Request]
Under _Site Settings > Layout > Layout Settings_, you can only enter a **Content Width** in pixels, which becomes challenging for responsiveness because you can easily end up with too little spacing at the edge of the screen on some devices.

I know we can add custom break points or CSS to achieve this, but I think there's an easier solution.

It would be awesome to have a new "Content Max-Width" setting that can use vw or %.

Example: I could set the Content Width on tablet to 1000px and then add a "Content Max Width" of 95% to account for screens where 1000px might not give enough breathing room at the edges of the screen.

**Comment by pauserratgutierrez** [comment]
Min-width should be added too.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the introduction of Custom Units in Elementor version 3.10, and improved with more units in Elementor version 3.12! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
